index.html

```diff
@@ -1916,9 +1943,9 @@ <h3 class="no-num no-ref heading settled" id="normative"><span class="content">N
    <ul>
     <li><a href="#ref-for-magnetometer-magnetometers-1">1. Introduction</a>
     <li><a href="#ref-for-magnetometer-magnetometers-2">4.1. Common fusion sensors</a> <a href="#ref-for-magnetometer-magnetometers-3">(2)</a> <a href="#ref-for-magnetometer-magnetometers-4">(3)</a> <a href="#ref-for-magnetometer-magnetometers-5">(4)</a>
-    <li><a href="#ref-for-magnetometer-magnetometers-6">4.3. Orientation Sensor</a>
-    <li><a href="#ref-for-magnetometer-magnetometers-7">4.4. Geomagnetic Orientation Sensor</a>
-    <li><a href="#ref-for-magnetometer-magnetometers-8">4.6. Gravity and Linear Acceleration Sensor</a>
+    <li><a href="#ref-for-magnetometer-magnetometers-6">4.3. Orientation Sensor</a> <a href="#ref-for-magnetometer-magnetometers-7">(2)</a>
+    <li><a href="#ref-for-magnetometer-magnetometers-8">4.4. Geomagnetic Orientation Sensor</a>
+    <li><a href="#ref-for-magnetometer-magnetometers-9">4.6. Gravity and Linear Acceleration Sensor</a>
    </ul>
   </aside>
   <aside class="dfn-panel" data-for="orientation-sensor">
```

Comment:
@kenchris I updated the illustration and text, could you check if I need to explain inclination of magnetic field vector.